tests/php/ORM/DataListEagerLoadingTest.php

```diff
@@ -1288,5 +1289,226 @@ public function testEagerLoadingSharedRelations()
                 $this->assertInstanceOf(EagerLoadedList::class, $eagerLoaded2);
             }
         }
+        $this->assertGreaterThan(1, $loop1Count);
+        $this->assertGreaterThan(1, $loop2Count);
```

Comment:
Unrelated change - the test was meant to have these originally which is why those variables were there.